scipy/stats/_continuous_distns.py

```diff
@@ -5925,7 +5935,8 @@ def _stats(self, dfn, dfd, nc):
 
 
 class t_gen(rv_continuous):
-    r"""A Student's t continuous random variable.
+    r"""A Student's t continuous random variable. For the noncentral
+    chi-square distribution, see `nct`.
```

Comment:
Split the line, and also correct the referenced name.
```suggestion
    r"""A Student's t continuous random variable.
    
    For the noncentral t distribution, see `nct`.
```